lib/qless/job.rb

```diff
@@ -227,8 +227,10 @@ def to_hash
 
     # Move this from it's current queue into another
     def requeue(queue, opts = {})
+      queue_name = queue.is_a?(String) ? queue : queue.name
```

Comment:
That would be a breaking change.  Historically, this method has always accepted (and expected) a string for the queue name.  As a result, if you called `job.requeue(queue_object)`, it would convert the queue to a string using `to_s` which would put the job onto a queue with a name like `#<Qless::Queue queue_name>`.  This is why we have had some jobs listed on the qless UI in a queue named `#` (the rest of the name is interpretted as a markup tag).  I made that mistake last night and enqueued 10 jobs to a queue with that name.  (I fixed it this morning).  The change here is to guard against that.